src/main/scala/uk/gov/nationalarchives/draftmetadatavalidator/ErrorFileData.scala

```diff
@@ -9,11 +10,27 @@ import java.util.{Date, UUID}
 
 object FileError extends Enumeration {
   type FileError = Value
-  val UTF_8, INVALID_CSV, SCHEMA_REQUIRED, DUPLICATE_HEADER, SCHEMA_VALIDATION, UNKNOWN, None = Value
+  val UTF_8, INVALID_CSV, ROW_VALIDATION, SCHEMA_REQUIRED, DUPLICATE_HEADER, SCHEMA_VALIDATION, UNKNOWN, None = Value
 }
 
 case class Error(validationProcess: String, property: String, errorKey: String, message: String)
 case class ValidationErrors(assetId: String, errors: Set[Error], data: List[Metadata] = List.empty[Metadata])
+
+object ValidationErrors {
+  implicit val validationErrorsListSemigroup: Semigroup[List[ValidationErrors]] =
```

Comment:
Shouldn't include the type in the `val` name. Maybe rename to something like this: `combineValidationErrors`?